homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html

```diff
@@ -32,12 +32,27 @@
                     <div class="return">
                         <button type="button" class="button">Calculation</button>
                     </div>
-                    <div class="re">
-                        <p class="resul__number result"></p>
+                    <div class="count-block">
+                        <div class="visual-counting">
+                            <div class="number__one">
+                                <h2 class="number__1"></h2>
+                            </div>
+                            <div class="operator__visual">
+                                <h2 class="oper"></h2>
```

Comment:
It is not a title. Title - is the semantic tag, important for the search machine to analyze your page. It is `<span>`.